backend/app/services/execution/dispatcher.py

```
@@ -728,6 +734,196 @@ async def _dispatch_websocket(
+
+            # Call ChatService.chat() which handles all the streaming internally
+            await chat_service.chat(request, responses_emitter)
+
+            logger.info(
+                f"[ExecutionDispatcher] In-process chat completed: "
+                f"task_id={request.task_id}, subtask_id={request.subtask_id}"
+            )
+        except Exception as e:
+            logger.exception(
+                f"[ExecutionDispatcher] In-process chat error: "
+                f"task_id={request.task_id}, error={e}"
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)